### PR TITLE
feat: fix withdraw all

### PR DESCRIPTION
**Pull request by Kodylow**
Adds withdraw all fixes from pairing with justin, requires fixes to fedimint.

### CODE REVIEW

**Comment by wbobeirne** [comment]
> requires fixes to fedimint.

Should we move this into draft? Is there an issue / PR where this is being tracked in fedimint?

**Comment by justinmoon** [comment]
"withdraw all" should be fixed on `releases/0.2` https://github.com/fedimint/fedimint/pull/3803

**Review by wbobeirne — APPROVED** [review]
Worked great off of rc5 from #337. Please hold off on merging until that gets merged though, want to make sure master reflects working state.

**Pull request merged**
